### PR TITLE
add Bumhan Yu (@baadaa) && fix build failure on `Person.js`

### PR DESCRIPTION
**Pull request by baadaa**

### CODE REVIEW

**Comment by baadaa** [comment]
Just a quick recap of the two commits in this PR

1. Adding my info based on the existing `data.js` structure
2. Adding a ternary conditional for Twitter string to avoid breaking the build when there's no `twitter` value

File: src/components/Person.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ function useIntersectionObserver(ref) {
 
 export default function Person({ person, currentTag }) {
   const url = new URL(person.url);
-  const twitter = `https://unavatar.now.sh/twitter/${person.twitter.replace('@', '')}`;
```
Comment (baadaa) [inline src/components/Person.js:27]: This string interpolation fails when it encounters a data entry with no `twitter` key in it—with the error `Cannot read property 'replace' of null`. My PR addresses it by a ternary operator to interpolate the string only when there's a `twitter` value.

**Comment by baadaa** [comment]
Just checking in @HugoDF & @wesbos — many of currently open PRs fail during build at the same point (i.e. `Person.js` line 27), and this PR addresses the fix. Could you please review and merge? I believe it'll help resolve many of the open PRs.

image

**Comment by HugoDF** [comment]
Thanks for fixing that

**Comment by mreduar** [comment]
@HugoDF Do we need to send a new PR?

**Pull request merged**